Source/DafnyTestGeneration/Utils.cs

```diff
@@ -16,6 +16,23 @@ namespace DafnyTestGeneration {
 
   public static class Utils {
 
+    /// <summary>
+    /// Call Translator with larger stack to prevent stack overflow
+    /// </summary>
+    public static List<Microsoft.Boogie.Program> Translate(Program program) {
+      var ret = new List<Microsoft.Boogie.Program> { };
+      var thread = new System.Threading.Thread(
+        () => {
+          ret = Translator
+    .Translate(program, program.Reporter)
```

Comment:
This is a tiny nit, but could indent this line and the following beyond the indentation level of `ret`?

Reply:
Done!